src/Microsoft.DocAsCode.Build.TableOfContents/BuildTocDocumentStepBase.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.TableOfContents
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Linq;
+
+    using Microsoft.DocAsCode.Build.Common;
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    public abstract class BuildTocDocumentStepBase : BaseDocumentBuildStep
+    {
+        internal ImmutableDictionary<string, TocItemInfo> TocModelCache = ImmutableDictionary<string, TocItemInfo>.Empty;
```

Comment:
how about pass it via parameter?

---
In reply to: [121076786](https://github.com/dotnet/docfx/pull/1750#discussion_r121076786) [](ancestors = 121076786)